shoov/modules/custom/shoov_repository/handlers/shoov_repository_subscribe_user.inc

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/**
+ * @file
+ * Contains shoov_repository_subscribe_user.
+ */
+
+class shoov_repository_subscribe_user extends views_handler_field_entity {
+
+  /**
+   * {@inheritdoc}
+   */
+  function render($values) {
+    // Get group is being viewed ID.
+    $argument = $this->view->argument;
+    if (empty($argument['og_user_node_target_id'])) {
+      // The context hasn't been sent.
+      return;
+    }
+    $gid = $argument['og_user_node_target_id']->value[0];
```

Comment:
this looks wrong-  you should probably use `getValue()` or some other method to get the value "the right way"